src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (a) => {
```

Comment:
The `adder` function does not check if the argument `a` is a number. According to the additional prompt, there is no need to check if the argument is a number, so this is acceptable. However, be aware that passing non-numeric values will result in unexpected behavior.